org.kde.kalm.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.kde.kalm
+runtime: org.kde.Platform
+runtime-version: 6.7
+sdk: org.kde.Sdk
+command: kalm
+copy-icon: true
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/libexec
+  - /mkspecs
+  - /etc/xdg/*categories
+  - /share/qlogging-categories5
+modules:
+  - name: kirigami-addons
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_TESTING=OFF
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.2.1.tar.xz
+        sha256: 657426f9ec55cfec5a7f04848ed2b4df2ac8bfcb33c25626ad239219d444ee98
+        x-checker-data:
+          type: anitya
+          project-id: 242933
+          stable-only: true
+          url-template: https://download.kde.org/stable/kirigami-addons/kirigami-addons-$version.tar.xz
+  - name: kalm
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/release-service/24.05.0/src/kalm-24.05.0.tar.xz
+        sha256: 55999d5872e7b4569f5c31b893f19f91a2fdf0d89a8ba4e672d9b69e45a1bab6
+        #x-checker-data:
+        #  type: anitya
+        #  project-id: TODO
```

Comment:
If this is part of KDE Apps/Gears then there is a common ID for all apps.

Reply:
It is. What's the ID?

Reply:
This: https://release-monitoring.org/project/8763/?

Reply:
yes